### PR TITLE
[WIP] docs: update documentation

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Comment by akkatracker** [comment]
Perhaps #17529 can be addressed in this same PR (a simple typo in a link within the contributing docs)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+2. 👀️ Follow the contributing guidelines outlined below.
+3. 🔧 Make some awesome changes!
+4. 👉 [Make a pull request](https://github.com/freeCodeCamp/freeCodeCamp/compare)
+5. 🎉 Get your pull request approved - success!
 
-3. Follow the instructions in this guide to start working on the issue.
+Or, If you've found a issue that is not on the board, [follow these steps](README.md#found-a-bug).
 
-Remember to feel free to ask for help in our [Contributors](https://gitter.im/FreeCodeCamp/Contributors) Gitter room.
+Feel free to ask for help in our public [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors). We are excited to help you contribute to any of the projects that you would like to work on.
```
Comment (Bouncey) [inline CONTRIBUTING.md]: Feel free to ask for help in our public [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors), or on our [forum](https://forum.freecodecamp.org/c/contributors). We are excited to help you contribute to any of the projects that you would like to work on.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -108,46 +90,59 @@ This will download the entire freeCodeCamp repo to your projects directory.
...
 1. Make sure you are on the `staging` branch
 
-```shell
-$ git status
-On branch staging
-Your branch is up-to-date with 'origin/staging'.
-```
-If your aren't on `staging`, resolve outstanding files / commits and checkout the `staging` branch
+    ```shell
+    $ git status
```
Comment (Bouncey) [inline CONTRIBUTING.md]: `git status`

With a separate code block for the expected output.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -387,28 +426,29 @@ For each challenge section, there is a JSON file (fields documented below) conta
...
 When you're ready to share your code, run the test suite:
 
 ```shell
-$ npm test
+npm run test
 ```
 
 and ensure all tests pass.
 
+if you see some tests failing you can run:
```
Comment (Bouncey) [inline CONTRIBUTING.md]: Unfinished sentence?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -425,11 +465,7 @@ related issues in our [Contributors chat room](https://gitter.im/FreeCodeCamp/Co
 
 #### Important: ALWAYS EDIT ON A BRANCH
 
-Take away only one thing from this document: Never, **EVER**
-make edits to the `staging` branch. ALWAYS make a new branch BEFORE you edit
-files. This is critical, because if your PR is not accepted, your copy of
-staging will be forever sullied and the only way to fix it is to delete your
-fork and re-fork.
+Take away only one thing from this document: Never, **EVER** make edits to the `staging` branch. ALWAYS make a new branch BEFORE you edit files. This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.
```
Comment (Bouncey) [inline CONTRIBUTING.md]: > the only way to fix it is to delete your fork and re-fork

Not true, you can `git reset --hard upstream/staging`, you will lose any commits outside of the upstream commit history, but there is no need to re-fork.

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (raisedadead) [reply CONTRIBUTING.md]: Totally true. This is some formatting WIP. I am going to remove all of this outdated git steps. These can be made lot simpler.

File: CONTRIBUTING.md (hunk 3 above)
Comment (raisedadead) [reply CONTRIBUTING.md]: WIP commit.

File: CONTRIBUTING.md (hunk 2 above)
Comment (raisedadead) [reply CONTRIBUTING.md]: Totally agree with you. This is some formatting WIP. I am going to remove all of this outdated git steps. These can be made lot simpler.

File: CONTRIBUTING.md (hunk 1 above)
Comment (raisedadead) [reply CONTRIBUTING.md]: Ah.. right we should add the form at all places. Thanks I'll update.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -425,11 +465,7 @@ related issues in our [Contributors chat room](https://gitter.im/FreeCodeCamp/Co
 
 #### Important: ALWAYS EDIT ON A BRANCH
```
Comment (QuincyLarson) [inline CONTRIBUTING.md]: Please don't use all caps. We are not Donald Trump.

**Review by Seilamuk610 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
 
-2. Let us know you are working on it by posting a comment on the issue.
+1. 🍴 [Fork this repo](https://github.com/freeCodeCamp/freeCodeCamp#fork-destination-box)
+2. 👀️ Follow the contributing guidelines outlined below.
+3. 🔧 Make some awesome changes!
+4. 👉 [Make a pull request](https://github.com/freeCodeCamp/freeCodeCamp/compare)
+5. 🎉 Get your pull request approved - success!
 
-3. Follow the instructions in this guide to start working on the issue.
+Or, If you've found a issue that is not on the board, [follow these steps](README.md#found-a-bug).
```
Comment (erictleung) [inline CONTRIBUTING.md]: Remove lowercase for "If"

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -38,68 +42,46 @@ Working on your first Pull Request? You can learn how from this *free* series [H
...
-2. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
+1. [Install Git](https://git-scm.com/) or your favorite Git client.
+2. [Setup a SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub. (Optional)
 
 #### Forking freeCodeCamp
 
-1. Go to the top level freeCodeCamp repository: <https://github.com/freeCodeCamp/freeCodeCamp>
-2. Click the "Fork" Button in the upper right hand corner of the interface ([More Details Here](https://help.github.com/articles/fork-a-repo/))
-3. After the repository (repo) has been forked, you will be taken to your copy of the freeCodeCamp repo at <https://github.com/yourUsername/freeCodeCamp>
+1. Go to the top level freeCodeCamp repository: [`https://github.com/freeCodeCamp/freeCodeCamp`](https://github.com/freeCodeCamp/freeCodeCamp)
```
Comment (erictleung) [inline CONTRIBUTING.md]: Hyphenate ["top-level"](https://english.stackexchange.com/questions/45945/how-does-the-hyphen-change-the-meaning-in-expressions-like-high-performance-an)? Or reword to "Go to the main freeCodeCamp repository"?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -157,30 +152,71 @@ Before you start working, you will need to create a separate branch specific to
...
+mongo --version
+```
+
+**Windows:**
+
+The installer for Node.js usually updates your path. So the same commands from the Linux/macOS setup should work.
+
+To check your MongoDB version on Windows, you have to locate the installation directory. It is probably located at something like `C:\Program Files\MongoDB\Server\3.6\` where `3.6` is your version number.
+
+You may also want to add it to your System Environment `PATH` variable if not already, to access CLI tools like `mongo`
```
Comment (erictleung) [inline CONTRIBUTING.md]: Missing period at the end of this sentence?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -213,40 +249,41 @@ First install Go.
...
+```shell
 go get github.com/mailhog/MailHog
 sudo cp /home/$(whoami)/go/bin/MailHog /usr/local/bin/mailhog
 mailhog
 ```
 
 #### Using MailHog
 
-Once you have installed MailHog and started it running you need to open your MailHog inbox in your browser, open a new tab or window and navigate to [http://localhost:8025](http://localhost:8025).
+Once you have installed MailHog and started it running you need to open your MailHog inbox in your browser, open a new tab or window and navigate to [`http://localhost:8025`](http://localhost:8025).
```
Comment (erictleung) [inline CONTRIBUTING.md]: Maybe reword to something like this (emphasis added for highlighting changes):

> Once you have installed MailHog and started it running, you need to open your MailHog inbox in your browser **by openning** a new tab or window and **navigating** to [`http://localhost:8025`](http://localhost:8025).

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -330,54 +368,55 @@ node seed
...
 To create a new user, you will need to perform the following steps:
-* run MailHog if you haven't set up yet check [set-up-mailhog](#set-up-mailhog)
-* run `npm run develop` and navigated to [http://localhost:3000](http://localhost:3000/)
-* Click over the Sign-Up link situated on the right corner of the navigation bar.
-* Write your email over the input and press the button "get a sign in link" this will send an email with the login URL into MailHog.
-* The last step is to check your inbox in MailHog for a new email from "team@freecodecamp.org" click over the URL inside of it you will be redirected and logged in into the app.
 
-### Make Changes
+- run MailHog if you haven't set up yet check [set-up-mailhog](#set-up-mailhog)
+- run `npm run develop` and navigated to [http://localhost:3000](http://localhost:3000/)
```
Comment (erictleung) [inline CONTRIBUTING.md]:
```diff
- ...and navigated to...
+ ...and navigate to...
```

And put local host in back ticks to be consistent with other mentions of local host.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -330,54 +368,55 @@ node seed
...
-#### How to find the code in the freeCodeCamp codebase to fix/edit
+### How to find the code in the freeCodeCamp codebase to fix/edit
 
-The best way to find out any code you wish to change/add or remove is using
-the GitHub search bar at the top of the repository page. For example, you could
-search for a challenge name and the results will display all the files along
-with line numbers. Then you can proceed to the files and verify this is the area
-that you were looking forward to edit. Always feel free to reach out to the chat
-room when you are not certain of any thing specific in the code.
+The best way to find out any code you wish to change/add or remove is using the GitHub search bar at the top of the repository page. For example, you could search for a challenge name and the results will display all the files along with line numbers. Then you can proceed to the files and verify this is the area that you were looking forward to edit. Always feel free to reach out to the chat room when you are not certain of any thing specific in the code.
```
Comment (erictleung) [inline CONTRIBUTING.md]: Last sentence "any thing" --> "anything"

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -387,28 +426,29 @@ For each challenge section, there is a JSON file (fields documented below) conta
 
 For more about creating challenges, see [seed/README](seed/README.md) and [seed/challenge-style-guide.md](seed/challenge-style-guide.md).
 
-#### Changes to the seed files
+### Changes to the seed files
+
+If you made changes to any file in the `/seed` directory, you then need to stop the server by typing CTRL-C, then you need to run:
```
Comment (erictleung) [inline CONTRIBUTING.md]: We could be fancy and use `<kbd>Ctrl-C</kbd>` to look like <kbd>Ctrl-C</kbd>

**Comment by erictleung** [comment]
@raisedadead thanks for taking time to update the docs! I've made some suggestions that might help re-edit them.

**Comment by raisedadead** [comment]
@erictleung thanks a lot for taking the time to review this. We have been refactoring the code to make it more mono repo like and easier to contribute to.

These docs will probably need a little more re-write at places once we are ready in the next couple weeks. I'll add your recommendations as well.

Thanks.

File: CONTRIBUTING.md (hunk 7 above)
Comment (nathfreder) [reply CONTRIBUTING.md]: "Setup **a** SSH Key" should be "Setup **an** SSH Key"

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Closing. Will re-open after restructuring.
